website/docs/cdktf/concepts/modules.mdx

```diff
@@ -87,6 +87,8 @@ For local modules, use the object format.
 }
 ```
 
+For performance reasons we don't automatically generate bindings for submodules. To generate bindings for submodules, specify the module source as `terraform-aws-modules/eks/aws//modules/eks-managed-node-group`, where after the `//` is the path to the submodule in the modules repository.
```

Comment:
Given the wording, it feels like a jump to go from the vpc module example above this to some completely different module. Might be better to use a related submodule (https://registry.terraform.io/modules/terraform-aws-modules/vpc/aws/latest/submodules/vpc-endpoints) or tweak the wording slightly to make it clear that this a different example.

Might also be nice to indicate that any module source format specified in https://www.terraform.io/language/modules/sources should work.